tests/data_model/mock_data/mock_item_q1_class.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Silly github :P.

`__init__.py` is still needed because the resource needs to be loaded from a python package. There is some *very* thick details here: https://docs.python.org/3/library/importlib.html#importlib.abc.ResourceReader

However, I just confirmed that deleting `__init__.py` makes the tests fail with: 
`FileNotFoundError: Package has no location <module 'tests.data_model.mock_data' (namespace)>`

Reply:
hmm, it's because you use pkg_resources and don't load the file directly. I haven't seen such usage but it's not a bad idea to have in general.
You can just load the data in MANIFEST
https://python-packaging.readthedocs.io/en/latest/non-code-files.html

But it doesn't matter TBH. Small nitpick.

Reply:
Thinking about it, it shouldn't be read from pkg_resources , you should read the file directly. The tests are not part of the package.

Reply:
They aren't part of the referenceisland package though? They are just part of the test package.

Would you prefer they were read by a relative path? I saw other similar suggestions using `os.join()` and `__file__`

Reply:
setup.py installs both and technically it shouldn't (on the grounds that test code should never touch production, keeping it as separate as possible is better)
yeah, relative path is fine IMO, mediawiki does it all the time, sometimes it's annoying when you move test files but that's hopefully rare.